client/my-sites/site-monitoring/metrics-tab.tsx

```diff
@@ -149,6 +149,38 @@ function useAggregateSiteMetricsData(
 	};
 }
 
+/**
+ * Filter the data to only include the listed HTTP verbs
+ * @param timeRange
+ * @param metric
+ * @param dimension
+ */
+function useAggregateHttpVerbData(
+	timeRange: TimeRange,
+	metric?: MetricsType,
+	dimension?: DimensionParams
+): { formattedData: Record< string, number > } {
+	const data: { formattedData: Record< string, number > } = useAggregateSiteMetricsData(
+		timeRange,
+		metric,
+		dimension
+	);
+	const allowedVerbs: string[] = [ 'GET', 'POST', 'HEAD', 'DELETE' ];
```

Comment:
Should we also add PUT which is widely used by REST API endpoints?

Reply:
I think it makes sense to add PUT and PATCH as they are both commonly used methods (in general, not on WordPress.com).